src/CoreBundle/Event/HtmlIncludeEvent.php

```diff
@@ -58,13 +58,22 @@ public function getData()
         return $this->data;
     }
 
+    public function removeDataElement(string $key): void
+    {
+        unset($this->data[$key]);
```

Comment:
unset does not produce an error when unsetting a key that doesn't exist, however: Is this the functionality we want? Does the user (or the program somewhere higher in the stack) need to know about a failed removeDataElement attempt?

Reply:
added boolen return value (this may be interesting for debugging if a key really existed or not)